File: 02-C/09-funcionesIncorporadas/02subcadena.c
#include <stdio.h>
#include <string.h>

int main()
{
    // Obtener subcadena
    char cadena[] = "Hola a todos";
    char subcadena[10];

    // Se copia a partir del caracter indicado de inicio e indicamos cuantos caracteres se copian.
    int inicio = 0, nCaracteres = 4;

    strncpy(subcadena, &cadena[inicio], nCaracteres);
    subcadena[nCaracteres];

    printf("Cadena original: %s\n", cadena);
    printf("Subcadena: %s\n", subcadena);

    return 0;
}

File: 04-CSHARP/09-FuncionesIncorporadas/02subCadena.cs
// SubCadenas en C#

string cadena = "Hola a todos"; 

// Obtener subcadenas 
// substring(indiceInicio, numeroCaracteres)
var subCadena = cadena.Substring(0, 4); 
Console.WriteLine($"Subcadena: {subCadena}");

var subCadena2 = cadena.Substring(7, 5);
Console.WriteLine($"Subcadena 2: {subCadena2}");